RPIPs/RPIP-28.md

```diff
@@ -0,0 +1,47 @@
+---
+rpip: 28
+title: Deposits Under the Minimum
+description: Allow minipool deposits that include an appropriate RPL stake, even when under the "minimum" RPL threshold 
+author: Valdorff (@Valdorff)
+discussions-to: 
+status: Draft
+type: Protocol
+category: Core
+created: 2023-08-11
+---
+
+## Abstract
+
+Right now, if an NO falls below the minimum (value of RPL staked <10% of borrowed ETH), they can't 
+add a new minipool to that node unless they stake enough to reach the minimum AND cover the new
+minipool. Unless, of course, they make a new node. This means that we're creating an incentive to
+make Sybil nodes.
+
+This proposal suggests adding the ability to do a combined minipool deposit AND RPL stake. This
+would allow creating a new minipool even when under the minimum and remove the incentive to create
+Sybil nodes. Because creating a Sybil node is an inconvenience, we can require _slightly_ more RPL
+to be staked, such that the NO moves towards the minimum overall.
+
+## Specification
+
+- There SHALL be a function that allows an atomic RPL stake and minipool ETH deposit
+- This function SHALL succeed if the staked RPL meets of the amount that would be needed for this
+  minipool on a fresh node
+  - Note that this means the minipool MAY be created even if the overall node's RPL stake is below
+    the minimum needed (eg, for RPL rewards) 
+
+## Reference Implementation
+
+- Add a new `RocketNodeDeposit.deposit_with_RPL` function that has the same arguments as
+  `RocketNodeDeposit.deposit` plus a new `rpl_to_stake` argument 
+- `RocketNodeDeposit.deposit` now calls `RocketNodeDeposit.deposit_with_RPL` with rpl_to_stake=0
+- In `RocketNodeDeposit.deposit_with_RPL`
+  - `require(msg.value == _bondAmount, "Invalid value");`
+  - `rpl_requirement_met == (rocketNetworkPrices.getRPLPrice() * rpl_to_stake > rocketDAOProtocolSettingsNode.getMinimumPerMinipoolStake() * (32-_bondAmount))`
+  - Pass along rpl_requirement_met as needed such that the deposit will succeed if EITHER:
+    - `rpl_requirement_met` is true
```

Comment:
maybe call it `this_deposit_enough_rpl` or something, to make it clear that it's only one disjunct of the overall RPL requirement?